src/targets/gpu/fuse_mlir.cpp

```diff
@@ -35,7 +36,9 @@ struct module;
 
 namespace gpu {
 
-#ifdef MIGRAPHX_MLIR
+MIGRAPHX_DECLARE_ENV_VAR(MIGRAPHX_MLIR);
```

Comment:
Should be called `MIGRAPHX_ENABLE_MLIR`.

Reply:
Ack